spring-boot-admin-server-ui/package.json

```diff
@@ -45,6 +45,7 @@
     "file-saver": "2.0.5",
     "fuse.js": "^7.0.0",
     "iso8601-duration": "2.1.2",
+    "js-yaml": "^4.1.0",
```

Comment:
JSON is a good fit for JavaScript, so lets use YAML for representation in frontend :D